spinalcordtoolbox/image.py

```diff
@@ -1854,28 +1854,30 @@ def check_image_kind(img):
 
         - 'seg': Binary segmentation (0/1)
         - 'softseg': Nonbinary segmentation in the range [0, 1], where 0 and 1 are the majority of values
-        - 'labeled_seg': Nonbinary, whole values (0, 1, 2...) or (0.0, 1.0, 2.0...) only
-        - 'im': Any other float-valued image where 0 and 1 are not the majority of values.
+        - 'labeled_seg': Nonbinary, whole values (0, 1, 2...) where 0 is the majority of values
+        - 'im': Any other image where 0 is not the majority of values.
```

Comment:
If you want to cover all the cases, I don't think you can say that the "default" case of `anat` has any guarantees about majority values. (Also, it's `anat` rather than `im`).
```suggestion
        - 'anat': Any other image.
```